Articles/Adding-metadata-to-tickets-created-by-the-Mobile-SDK.md

```diff
@@ -0,0 +1,124 @@
+	ZDKCustomField *customFieldDeviceFreeSpace = [[ZDKCustomField alloc] initWithFieldId:@(00000000) andValue:deviceFreeSpace];
+ 
+	//Device battery level
+	NSString *deviceBatteryLevel = [NSString stringWithFormat:@"%f", [ZDKDeviceInfo batteryLevel]];
+	ZDKCustomField *customFieldDeviceBatteryLevel = [[ZDKCustomField alloc] initWithFieldId:@(00000001) andValue:deviceBatteryLevel];
+
+	[ZDKConfig instance].customTicketFields = @[customFieldDeviceFreeSpace, customFieldDeviceBatteryLevel];
+	
+```
+#### Android
+```java
+
+	String appVersion = String.format(Locale.US, "Version %s", BuildConfig.VERSION_NAME);
+	CustomField customFieldAppVersion = new CustomField(0l, appVersion);
+
+	String osVersion = String.format(Locale.US, "Android %s, Version %s", VERSION.RELEASE, VERSION.SDK_INT);
+	CustomField customFieldOsVersion = new CustomField(1l, osVersion);
+
+	ZendeskConfig.INSTANCE.setCustomFields(Arrays.asList(customFieldAppVersion, customFieldOsVersion));
+	
+```
+
+### Tags
+Tags are a very simple way to give context to and organize tickets. 
+#### iOS
+Tags are set by calling ```ZDKRequests configure:``` and setting the ```tags``` property of the RequestCreationConfig argument:
+
+```objective-c
+ 	[ZDKRequests configure:^(ZDKAccount *account, ZDKRequestCreationConfig *requestCreationConfig) {
+   		requestCreationConfig.tags = @[@"Example","iOS","paid_user"];
+	}
+```
+
+#### Android
+```java
+	ZendeskConfig.INSTANCE.setContactConfiguration(new ZendeskFeedbackConfiguration() {
+ 		@Override
+  		public List<String> getTags() {
+    		return Arrays.asList("Example", "Android", "paid_user");
+  		}
+	});
+
+```
+
+### Additional Request Information
+
+Additional request information allows you to append text to the body of every ticket a user submits using the SDK. Please note that this information will be visible by the end user when they view their ticket after submission.
+
+### iOS
+
+Again this is set by calling ```ZDKRequests configure:``` and setting the ```additionalRequestInfo``` property of the RequestCreationConfig argument:
+
+```objective-c
+	[ZDKRequests configure:^(ZDKAccount *account, ZDKRequestCreationConfig *requestCreationConfig) {
+    	requestCreationConfig.additionalRequestInfo = @"This will be at the bottom of every ticket";
+	}	
+```
+
+#### Android
+```java
+
+	ZendeskConfig.INSTANCE.setContactConfiguration(new ZendeskFeedbackConfiguration() {  
+  		@Override
+  		public String getAdditionalInfo() {
+    		return "This will be at the bottom of every ticket";
+  		}
+  
+  		@Override
+  		public String getRequestSubject() {
+    		return "Request Subject ";
+  		}
+	});
+
+```
+
+### Ticket Forms
+
+If your Zendesk instance supports custom forms you can specify the form that will be shown along with the ticket in the agent interface. The ticket form ID can be found in your Zendesk under Admin -> Ticket Forms (Under the Manage subheading). Simply click on the relevant form and copy the ID from the URL. 
```

Comment:
Good point, I'll add in a bit making it clear none of this will effect the ticket creation UI or UI in general.